instrumentation/kotlin-coroutines/kotlin-coroutines.gradle

```diff
@@ -0,0 +1,21 @@
+ext {
+  // TODO (trask) currently this is only needed for LambdaGen
+  //  revisit if we don't end up dropping Java 7
+  minJavaVersionForTests = JavaVersion.VERSION_1_8
+}
+
+apply from: "$rootDir/gradle/instrumentation.gradle"
+
+muzzle {
+  pass {
+    extraDependency('org.jetbrains.kotlin:kotlin-stdlib-common:1.3.72')
+    extraDependency('org.jetbrains.kotlinx:kotlinx-coroutines-core:1.3.7')
+    coreJdk()
```

Comment:
Since we're not instrumenting the JDK we shouldn't need this. We want muzzle to run against kotlin-coroutines versions so that also should be the target rather than an extra dependency. The normal muzzle pattern looks something like this, which is verifying the implementation applies to the range of versions of the library (we're interested in kotlin-coroutine-core).

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/master/instrumentation/jetty-8.0/jetty-8.0.gradle#L5

Reply:
I took a first pass at this; I'd appreciate feedback on how far back to go in the testing - I set it as `[1.0.0,)` since that's what made sense to me.

Reply:
Looking into muzzle failures here it appears that recent versions of kotlin-core offer a large number of variants.  I can't figure out how to get variant selection to work with muzzle; I've tried adding attributes to the `DefaultArtifact`'s `Properties`-based constructor, but that didn't seem to help.  To unblock the build I'm adding a range limit that doesn't include latest (1.3.9) and will circle back to this later.